.github/workflows/main.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       matrix:
         include:
           - flutter_version: dev
-            allow_failure: true
+            allow_failure: false
```

Comment:
With all three set to false, there's no need for this matrix variable any more. You can just take it out.